docs/_docs/installation.md

```diff
@@ -28,6 +29,14 @@ Before you start, make sure your system has the following:
   </p>
 </div>
 
+<div class="note info">
```

Comment:
Do we really need three notes in a row?
Is there a better way to present this?

Reply:
Agreed. Let's do a section per system.